peachjam/js/components/LegislationTable/index.vue

```diff
@@ -122,30 +125,50 @@
               <div class="column">
                 {{ row.citation }}
               </div>
-            <!-- TODO: Establish content requirement for sublegs, then implement accordion syntax -->
-            <!--            <div-->
-            <!--              v-if="row.sublegs"-->
-            <!--              :id="`row-accordion-${index}`"-->
-            <!--              class="accordion-collapse collapse column__accordion accordion"-->
-            <!--              data-bs-parent=".legislation-table__row"-->
-            <!--            >-->
-            <!--              <div class="accordion-body">-->
-            <!--                <div-->
-            <!--                  v-for="(leg, subleg_index) in row.sublegs"-->
-            <!--                  :key="subleg_index"-->
-            <!--                >-->
-            <!--                  {{ leg.title }}-->
-            <!--                </div>-->
-            <!--              </div>-->
-            <!--            </div>-->
+              <div
+                v-if="row.children.length"
+                :id="`row-accordion-${index}`"
+                class="accordion-collapse collapse accordion"
+                data-bs-parent=".legislation-table__row"
+              >
+                <div class="d-flex">
+                  <div class="indent" />
+                  <div class="accordion-body flex-grow-1">
+                    <div class="row">
```

Comment:
Since the columns and the info being presented are the same as the parent table -- title and citation -- I think these are redundant and can be removed.

Reply:
done